README.md

```diff
@@ -1,70 +1,220 @@
+<!-- > Add a link to your video project. -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<a href="https://www.loom.com/share/15131936398e4c0397663968e4bc0e8c?sid=3244ce95-7795-46cf-aea9-afdbd3e0828c">Get an understanding of the project by watching this video</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+### Prerequisites
+
+To get a local copy up and running, follow these steps.
+
+In order to run this project you need:
+
+- A text editor like Microsoft Visual Studio Code.
+- A Github account.
+- Internet Connection.
+- Basic HTML, CSS, JavaScript and React knowledge.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Bifabig/react-redux-capstone.git
+```
+
+### Install
+
+Run this command inside the cloned repository
+
+```sh
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+For unit testing
+
+```sh
+  npm test
+```
+
+For styling
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+For JavaScript
+
+```sh
+  npx eslint "**/*.{js,jsx}"
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+<a href="https://render.com">render</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<!-- > Mention all of the collaborators of this project. -->
+
+👤 **Biftu Girma**
+
+- GitHub: [@Bifabig](https://github.com/Bifabig)
+- Twitter: [@biftu94](https://twitter.com/biftu94)
+- LinkedIn: [Biftu Girma](https://www.linkedin.com/in/biftu-girma-3598a8126/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add User Login**
+- **Preserve data locally**
+
+<!-- > Mention future features. -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project:thumbsup:
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- > Give credit to everyone who inspired your codebase. -->
+
+Thanks to Microverse for this amazing project and Nelson Sakwa on Behance.
+
```

Comment:
- [ ] Nice job 👏 , However, kindly add the [appropriate link](https://www.behance.net/sakwadesignstudio) associated with the author 👍